pilot/pkg/proxy/envoy/proxy.go

```diff
@@ -50,12 +50,15 @@ type envoy struct {
 }
 
 // NewProxy creates an instance of the proxy control commands
-func NewProxy(config meshconfig.ProxyConfig, node string, logLevel string, pilotSAN []string, nodeIPs []string) proxy.Proxy {
+func NewProxy(config meshconfig.ProxyConfig, node string, logLevel string, componentLogLevel string, pilotSAN []string, nodeIPs []string) proxy.Proxy {
```

Comment:
nit: this list is getting long.  Maybe we should create a config object so that the parameter list is more clear?